src/style.css

```diff
@@ -1 +1,242 @@
-/* empty source */
+body {
+  --card-max-width: 198px;
+  --card-border: var(1px);
+  --card-border-radius: 5px;
+  --min-section-gap: 16px;
+  --min-section-top-gap: 16px;
+  --card-header-top-gap: 32px;
+  --card-margin: 0;
+  --card-padding: 0;
+  --card-padding-top: 40px;
+  --card-image-height: 134px;
+  --card-font-size: 12px;
+  --cardfont-color: #060b35;
+}
+
+article {
+  max-width: var(--card-max-width);
+  background-color: #fff;
+  border: var(--card-border) solid #f3f3f3;
+  border-radius: var(--card-border-radius);
+}
+
+header {
+  margin: var(--card-margin);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-left: var(--min-section-gap);
+  padding-right: var(--min-section-gap);
+  padding-top: var(--card-header-top-gap);
+}
+
+main {
+  display: block;
+}
+
+.card__image {
+  height: var(--card-image-height);
+  margin: var(--card-margin);
+  padding: var(--card-padding);
+}
+
+/* .title semantic tag {
+  padding: 0;
+  margin: 0;
+  margin-top: 40px;
+  padding-right: 16px;
+  padding-left: 16px;
+  display: flex;
+  justify-content: center;
+  text-align: flex-start;
+  align-content: top;
+} */
+
+.card__title {
+  margin: var(--card-margin);
+  vertical-align: text-top;
+  padding-left: var(--min-section-gap);
+  padding-right: var(--min-section-gap);
+  padding-top: var(--card-padding-top);
+  font-size: var(--card-font-size);
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  font-style: medium;
+  line-height: 18px;
+  color: var(--cardfont-color);
+}
+
+/* .description {
+  margin: 0;
+
+  margin-right: 84px;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+} */
+
+.card__description {
+  margin: 0;
+  padding: 0;
+  padding-top: 4px;
+  padding-left: var(--min-section-gap);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+/* .distance1 :nth-child(1) {
+  display: flex;
+  align-items: center;
+  justify-content: left;
+}
+
+.distance1 :nth-child(2) {
+  display: flex;
+  align-items: bottom;
+  justify-content: right;
+} */
+
+section {
+  padding: var(--card-padding);
+  margin: var(--card-margin);
+  padding-left: var(--min-section-gap);
+  padding-top: var(--min-section-top-gap);
+  padding-right: var(--min-section-gap);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.stars {
+  margin: var(--card-margin);
```

Comment:
You shouldn't set margins for the BEM block. 
If you copy the stars from here and try to use them somewhere else, then they will have margins inside.
It is not right behaviour. Block must be independent and easily used all over the project.
You can add class `card__stars` and move all the external positioning there, as an element of card block